pkg/services/librarypanels/models.go

```diff
@@ -0,0 +1,42 @@
+package librarypanels
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+)
+
+// LibraryPanel is the model for library panel definitions.
+type LibraryPanel struct {
+	Id       int64
+	OrgId    int64
```

Comment:
Following [this](https://github.com/grafana/grafana/pull/29642/files#r538102765) discussion if we want to conform with GO's case for initialisms this can become:
```suggestion
	ID        int64 `xorm:"pk autoincr 'id'"`
	OrgID     int64 `xorm:"org_id"`
```

Reply:
Thank you Sofia, I'll look into this

Reply:
I've fixed this in my latest commit (I hope)